sass/elements/_schedule.scss

```diff
@@ -0,0 +1,90 @@
+.schedule-blocks-regular {
+    background-color: $primary-color;
+    a {
+        color: $off-white-color;
+    } 
+    &:hover {
+        background-color: tint($primary-color, 60%);
+    } 
+}
+
+.schedule-blocks-sustainer {
+    background-color: $secondary-color;
+    color: $off-white-color;
+}
+
+.schedule-blocks-flagship {
+    background-color: $gold;
```

Comment:
Might be able to write a mixin that reduces a lot of the very similar code here, where only the var for bg-color and text-color change, but less important